falcon/response.py

```diff
@@ -108,15 +108,17 @@ class Response(object):
 
         stream_len (int): Deprecated alias for :py:attr:`content_length`.
 
-        context (dict): Dictionary to hold any data about the response which is
-            specific to your app. Falcon itself will not interact with this
-            attribute after it has been initialized.
+        context (object): Empty object to hold any data (in its attributes)
```

Comment:
I think it would be helpful to inline an example here to illustrate using attributes in lieu of the (dict) mapping interface.

Reply:
Same question as: https://github.com/falconry/falcon/pull/1408/files#r251250026